src/parser/core/ParseResults.tsx

```diff
@@ -339,13 +340,13 @@ export type When = <T extends ValidThresholds>(threshold: T) => GenericSuggestio
 class ParseResults {
   tabs: ParseResultsTab[] = [];
   statistics: React.ReactElement[] = [];
-  issues: Issue[] = [];
+  issues: Array<Issue | SuggestionData> = [];
```

Comment:
What's with the move from `[]` to `Array<>` - I'm not too familiar with the pro/con

Reply:
Simple types use `[]` and complex types `Array<x>`. Eslint will yell about it.